sdk-generate-entity-models-maven-plugin/src/main/java/com/hpe/adm/nga/sdk/generate/GeneratorHelper.java

```diff
@@ -56,12 +56,18 @@ public static String camelCaseFieldName(final String fieldName, boolean theFirst
         int wordCounter = 0;
         for (final String splitField : splitFields) {
             wordCounter++;
-            if (wordCounter > 1 || theFirstLetterIsCapital) {
-                stringBuffer.append(splitField.substring(0, 1).toUpperCase());
-            } else {
-                stringBuffer.append(splitField.substring(0, 1).toLowerCase());
+            try {
+                if (wordCounter > 1 || theFirstLetterIsCapital) {
+                    stringBuffer.append(splitField.substring(0, 1).toUpperCase());
```

Comment:
WordUtils.capitalize is available from commons lang, but this will probably ruin the catch exception you have there